### PR TITLE
Merging RT deadline and instructions changes into main

### PR DESCRIPTION
**Pull request by sbruinsma1**

### CODE REVIEW

File: src/js/outcome.js [hunk 1]
```diff
@@ -39,16 +39,26 @@ class Position {
   }
   trial(display_element, trial) {
     const show_circle = () => {
+      // check if make prediction (determines what feedback receive)
+      var predicExist = this.jsPsych.data.get().select('prediction').values.slice(-1)[0];
```
Comment (rythorpe) [inline src/js/outcome.js:43]: Is the goal to prevent the shield and picker from from rendering at a null position?

**Comment by rythorpe** [comment]
Looks good @sbruinsma1, minus a few small questions/comments. Once resolved, let's use the "Squash and merge" functionality so that everything gets combined into a single commit on main for simplicity. Does that sound alright to you?

File: src/js/outcome.js (hunk 1 above)
Comment (sbruinsma1) [reply src/js/outcome.js:43]: yes so if there's no response/null, there'll be no shield/picker

File: src/js/prediction.js [hunk 2]
```diff
@@ -120,6 +130,8 @@ class Click {
         prediction: response.prediction,
       };
 
+      // console.log('predic',trial_data.prediction)
+
```
Comment (rythorpe) [inline src/js/prediction.js:134]: Oh, here's another one.

**Comment by rythorpe** [comment]
Also, I just tested the task out and it seems to return hits and misses on null trials. I guess js handles null values as 0? Could we add a line of logic in [`assessPerformance`](https://github.com/sbruinsma1/adLearning_web_HC/blob/b0358fae31f545f599500bf7c096c1cd008727cd/src/js/blocksetting123.js#L133) to return `null` if `prediction==null`?

File: src/js/prediction.js (hunk 2 above)
Comment (rythorpe) [reply src/js/prediction.js]: Nevermind, fixed in https://github.com/sbruinsma1/adLearning_web_HC/pull/7/commits/5f7c9b02d63940a3fc160dff53bb94c4ff41965e.
